src/elements/form/Control.ts

```diff
@@ -95,11 +95,15 @@ import { Helpers } from './../../utils/Helpers';
                     <novo-quick-note *ngSwitchCase="'quick-note'" [formControlName]="control.key" [placeholder]="control.placeholder" [config]="control.config" (change)="modelChange($event)"></novo-quick-note>
                 </div>
             </div>
+            <div class="text-field-character-limit" [attr.data-automation-id]="text-area-character-count" *ngIf="_focused && control.maxlength && (control.controlType=='text-area' || control.controlType=='textbox')">
```

Comment:
I talked to mary a bit more about this and left some comments on the ticket based on that 9 I will link it for reference), but I will look into why the character count is in the middle, cause its not supposed to be there. https://jira.bullhorn.com/browse/BH-46921